include/flatbuffers/verifier.h

```diff
@@ -25,15 +25,27 @@ namespace flatbuffers {
 // Helper class to verify the integrity of a FlatBuffer
 class Verifier FLATBUFFERS_FINAL_CLASS {
  public:
+  struct Options {
+    uoffset_t max_depth = 64;
+    uoffset_t max_tables = 1000000;
+    bool check_alignment = true;
+    bool check_nested_flatbuffers = true;
+  };
+  Verifier(const uint8_t *const buf, const size_t buf_len, Options opts)
+      : Verifier(buf, buf_len, opts.max_depth, opts.max_tables,
+                 opts.check_alignment, opts.check_nested_flatbuffers) {}
+
   Verifier(const uint8_t *const buf, const size_t buf_len,
```

Comment:
Can we add a "deprecated" comment on this?

Reply:
Done